axelor-project/src/main/resources/views/Charts.xml

```diff
@@ -66,4 +66,16 @@
       widget="Progress"/>
     <field name="budgetedTime" title="Budgeted time" aggregate="sum" type="String"/>
   </grid>
+
+  <grid name="project-planning-time-dashboard-grid" title="Project"
+    model="com.axelor.apps.project.db.ProjectPlanningTime">
+    <field name="project.name" type="com.axelor.apps.project.db.Project"/>
+    <field name="employee.name" title="Employee" type="com.axelor.apps.hr.db.Employee"/>
+    <field name="projectTask.name" title="Project task"
+      type="com.axelor.apps.project.db.ProjectTask"/>
+    <field name="timeUnit.name" title="Time unit" type="String"/>
+    <field name="plannedTime" title="Planned time" aggregate="sum" type="String"/>
```

Comment:
Can you add this because planned time are only added in HR

if="__config__.app.isApp('employee') &amp;&amp; __config__.app.getApp('project')?.enablePlanification" if-module="axelor-human-resource"